daemons/dss-events-scribe/app.py

```diff
@@ -0,0 +1,72 @@
+"""
+See Readme.md in this directory for documentation on the dss-notify-v2 daemon.
+
+storage_event -> invoke_notify_daemon -> invoke_sfn -> sfn_dynamodb_loop -> sqs -> invoke_notify_daemon
+"""
+
+import os
+import sys
+import logging
+import typing
+import time
+from itertools import cycle
+from collections import namedtuple
+
+import domovoi
+from flashflood import FlashFloodJournalingError
+
+pkg_root = os.path.abspath(os.path.join(os.path.dirname(__file__), 'domovoilib'))  # noqa
+sys.path.insert(0, pkg_root)  # noqa
+
+import dss
+from dss import Config, Replica
+from dss.logging import configure_lambda_logging
+from dss.events import journal_flashflood, update_flashflood
+
+from dss.util import countdown
+
+
+configure_lambda_logging()
+logger = logging.getLogger(__name__)
+dss.Config.set_config(dss.BucketConfig.NORMAL)
+
+app = domovoi.Domovoi()
+
+
+class ReplicaStatus:
+    prefix = None
+    finished_journaling = False
+    finished_updating = False
+
+    def __init__(self, prefix):
+        self.prefix = prefix
+
+
+@app.scheduled_function("rate(5 minutes)")
+def flashflood_journal_and_update(event, context):
+    # TODO: Make this configurable
+    minimum_number_of_events = 10 if "dev" == os.environ['DSS_DEPLOYMENT_STAGE'] else 1000
+
+    def lambda_seconds_remaining():
+        # lambda time to live is configured with `lambda_timeout` in `daemons/dss-events-scribe/.chalice/config.json`
+        return context.get_remaining_time_in_millis() / 1000
```

Comment:
Nit: I think this returns a float, and I'd expect an `int` from the function name?  Would it be worth adding typing to this file?

Reply:
typing added